app/controllers/refinery/authentication/devise/admin/users_controller.rb

```diff
@@ -37,17 +37,17 @@ def edit
           def update
             # Store what the user selected.
             @selected_role_names = params[:user].delete(:roles) || []
-            @selected_role_names = @user.roles.select(:title).map(&:title) unless user_can_assign_roles?
+            @selected_role_names = @user.roles.pluck(:title) unless user_can_assign_roles?
             @selected_plugin_names = params[:user][:plugins]
 
             if user_is_locking_themselves_out?
-              flash.now[:error] = t('lockout_prevented', :scope => 'refinery.authentication.devise.admin.users.update')
-              render :edit and return
+              flash.now[:error] = t('lockout_prevented', scope: 'refinery.authentication.devise.admin.users.update')
+              render :edit && return
             end
 
             store_user_memento
 
-            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+            @user.roles = @selected_role_names.map(&:underscore)
```

Comment:
this was supposed to turn the array into roles

```suggestion
              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```